gen/aircraft.py

```diff
@@ -1473,6 +1497,32 @@ def register_special_waypoints(self, targets) -> None:
             if self.group.units[0].unit_type == F_14B and i == 0:
                 self.group.add_nav_target_point(t.position, "ST")
 
+#todo beide punkte sind circle?
```

Comment:
This is the difference between orbit and race track. Orbit needs only one point (the center). Orbit makes sense for AWACS, and tankers (which it sounds like you're thinking of doing next) will need to be race tracks. Will leave some more notes in flightplan.py to explain more.

Reply:
Good point. It makes more sense for the tanker to be on a racetrack instead of a circle point